src/VoiceProtocol.h

```diff
@@ -0,0 +1,47 @@
+// Copyright 2005-2020 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_VOICEPROTOCOLTYPE_H
```

Comment:
Include-guard does not (exactly) match file-name

Reply:
I must be lost during numerous refactors...